frontend/javascript/components/docupload/file-document.vue

```diff
@@ -183,15 +224,25 @@ import { DocumentMixin } from './lib/document_utils'
 
 import FileReview from './file-review.vue'
 import PdfPreview from './pdf-preview.vue'
+import FileBasicOperations from './file-basic-operations.vue'
 
 export default {
   name: 'FileDocument',
   components: {
     FileReview,
-    PdfPreview
+    PdfPreview,
+    FileBasicOperations
   },
   mixins: [I18nMixin, DocumentMixin],
-  props: ['config', 'document'],
+  props: [
+    'config',
+    'document',
+    'hideSelection',
+    'hideAdvancedOperations',
+    'iconStyle',
+    'showBasicOperations',
+    'highlightRedaction'
```

Comment:
do we need these props to support both the old and the new flow simultaneously? looks like these are very specific props that might overcomplicate the component

Reply:
agree, i will address this in the next pass, essentially replacing the "old style" components